ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/140__service_mesh.robot

```diff
@@ -0,0 +1,82 @@
+*** Settings ***
+Documentation       Test Cases to verify Service Mesh integration
+
+Library             Collections
+Resource            ../../../../Resources/OCP.resource
+Resource            ../../../../Resources/ODS.robot
+Suite Setup         Suite Setup
+Suite Teardown      Suite Teardown
+
+
+*** Variables ***
+${OPERATOR_NS}                              ${OPERATOR_NAMESPACE}
+${RHOAI_OPERATOR_DEPLOYMENT_NAME}           rhods-operator
+${DSCI_NAME}                                default-dsci
+${SERVICE_MESH_OPERATOR_NS}                 openshift-operators
+${SERVICE_MESH_CR_NS}                       istio-system
+${SERVICE_MESH_OPERATOR_DEPLOYMENT_NAME}    istio-operator
+
+${IS_PRESENT}                           0
+${IS_NOT_PRESENT}                       1
+
+
+*** Test Cases ***
+Validate Service Mesh State Managed
+    [Documentation]    The purpose of this Test Case is to validate Service Mesh integration
+    [Tags]    Operator    Smoke    ODS-2526    ServiceMesh-Managed
+
+    Is Resource Present    ServiceMeshControlPlane    data-science-smcp    ${SERVICE_MESH_CR_NS}    ${IS_PRESENT}
+    Check If Pod Exists    istiod    ${SERVICE_MESH_CR_NS}
+
+Validate Service Mesh State Unmanaged
+    [Documentation]    The purpose of this Test Case is to validate Service Mesh state 'Unmanaged'.
+    ...                The operator will not recreate/update the Service Mesh CR if removed or changed.
+    [Tags]    Operator    Smoke    ODS-2526    ServiceMesh-Unmanaged
+
+    Set Service Mesh Management State    Unmanaged    ${OPERATOR_NS}
+    Delete Service Mesh Control Plane    ${SERVICE_MESH_CR_NS}
+    Wait Until Keyword Succeeds    2 min    0 sec
+    ...    Is Resource Present    ServiceMeshControlPlane    data-science-smcp    ${SERVICE_MESH_CR_NS}    ${IS_NOT_PRESENT}
+
+    [Teardown]    Set Service Mesh Management State    Managed    ${OPERATOR_NS}
+
+Validate Service Mesh State Removed
+    [Documentation]    The purpose of this Test Case is to validate Service Mesh state 'Removed'.
+    ...                The operator will Delete the Service Mesh CR, when state is Removed.
+    ...                Test will fail until RHOAIENG-2209 is fixed
+    [Tags]    ServiceMesh-Removed
```

Comment:
shouldn't this one have same Tags as above test cases?

Reply:
@mattmahoneyrh why are you using different tag on this Test Case?

Reply:
@bdattoma @asanzgom  For the time being, this test case is not run with 'smoke' suite test because of RHOAIENG-2209. Lets not run tests with known failures as not to waist analysis time. Once the bug is fixes the tag will be changed appropriately.